Include/cpython/pyhash.h

```diff
@@ -10,14 +10,19 @@
    reduction modulo the prime 2**_PyHASH_BITS - 1. */
 
 #if SIZEOF_VOID_P >= 8
-#  define _PyHASH_BITS 61
+#  define PyHASH_BITS 61
 #else
-#  define _PyHASH_BITS 31
+#  define PyHASH_BITS 31
 #endif
 
-#define _PyHASH_MODULUS (((size_t)1 << _PyHASH_BITS) - 1)
-#define _PyHASH_INF 314159
-#define _PyHASH_IMAG _PyHASH_MULTIPLIER
+#define PyHASH_MODULUS (((size_t)1 << _PyHASH_BITS) - 1)
+#define PyHASH_INF 314159
+#define PyHASH_IMAG _PyHASH_MULTIPLIER
+
+#define _PyHASH_BITS PyHASH_BITS
```

Comment:
```suggestion
// Aliases kept for backward compatibility with Python 3.12
#define _PyHASH_BITS PyHASH_BITS
```

Reply:
Done, but using ``/* style */`` for consistency with the rest of file.